contributing.md

```diff
@@ -24,6 +25,10 @@ If you want to make a bigger change, it's a good idea to first file an issue and
 - We use [roxygen2](https://cran.r-project.org/web/packages/roxygen2/index.html) for basic documentation, [rmarkdown](https://rmarkdown.rstudio.com/docs/) for vignettes and [pkgdown](https://pkgdown.r-lib.org/) for websites.
 - We use [testthat](https://cran.r-project.org/web/packages/testthat/index.html) for unit tests. Contributions with test cases included are easier to accept.
 
+### Use cases
+
+We are always interested in hearing about how epiverse-trace packages are being applied. If you have a use case which you would like to share with the wider community, please do let us know. You can post about it in the discussions board of the relevant repository or reach out through [email](anna.carnegie@lshtm.ac.uk). We;re grateful for any way that you can spread the word. Whether that's citing epiverse-trace packages in your papers or telling your friends about a feature you found useful.
```

Comment:
```suggestion
We are always interested in hearing about how epiverse-trace packages are being applied. If you have a use case which you would like to share with the wider community, please do let us know. You can post about it in the discussions board of the relevant repository or reach out through [email](anna.carnegie@lshtm.ac.uk). We're grateful for any way that you can spread the word. Whether that's citing epiverse-trace packages in your papers or telling your friends about a feature you found useful.
```

Would it be worth mentioning why we want to collect use cases? Saying that they inform future development priorities by identifying which features are the most used, and which part of the project lack clarify or need improvement.

Reply:
Good suggestion. I've incorporated this wording.